qa/integration-tests/src/test/java/io/camunda/zeebe/it/system/BrokerAdminServiceClusterTest.java

```diff
@@ -73,10 +73,10 @@ public void shouldReportPartitionStatusOnFollowersAndLeader() {
     followerStatus.forEach(
         partitionStatus -> {
           assertThat(partitionStatus.getRole()).isEqualTo(Role.FOLLOWER);
-          assertThat(partitionStatus.getProcessedPosition()).isNull();
+          assertThat(partitionStatus.getProcessedPosition()).isEqualTo(-1L);
           assertThat(partitionStatus.getSnapshotId()).isNull();
           assertThat(partitionStatus.getProcessedPositionInSnapshot()).isNull();
-          assertThat(partitionStatus.getStreamProcessorPhase()).isNull();
+          assertThat(partitionStatus.getStreamProcessorPhase()).isEqualTo(Phase.REPLAY);
```

Comment:
Does it make sense that the changes made result in the Stream Processor Phase of the followers changing from null to Phase.REPLAY?

Reply:
Yes, that makes sense. In `PartitionStatus#ofFollower` this was explicitely set to `null`, but now we are actually asking the StreamProcessor for the phase. On followers this should always be `REPLAY` instead of `PROCESSING` which only leaders can do.